src/main/java/service/operation/ReturnOperation.java

```diff
@@ -0,0 +1,22 @@
+package service.operation;
+
+import data.db.Storage;
+import java.util.Map;
+import model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+
+    @Override
+    public void operation(FruitTransaction.Operation operation, FruitTransaction fruitTransaction) {
+        if (fruitTransaction.getQuantity() <= 0) {
+            return;
+        }
+        for (Map.Entry<String, Integer> fruit : Storage.fruitsStorage.entrySet()) {
+            if (fruit.getKey().equals(fruitTransaction.getFruit())) {
+                fruit.setValue(fruit.getValue() + fruitTransaction.getQuantity());
+                return;
+            }
+        }
+        Storage.fruitsStorage.put(fruitTransaction.getFruit(), fruitTransaction.getQuantity());
+    }
```

Comment:
It seems like the ReturnOperation class is directly manipulating the Storage.fruitsStorage. This is not following the encapsulation principle. Instead of accessing the storage directly, consider creating a service that will handle the interaction with the storage, and inject that service into your operation handlers.